src/main/java/ch/admin/bag/covidcertificate/sdk/core/utils/DateTimeExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package ch.admin.bag.covidcertificate.sdk.core.utils
+
```

Comment:
this should probably be in the extension package and not in the utils package.

Reply:
done